### PR TITLE
Swagger and CSPs

### PR DESCRIPTION
**Pull request by istreeter**

### CODE REVIEW

**Comment by snowplowcla** [comment]
Thanks for your pull request. Is this your first contribution to a Snowplow open source project? Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

:memo: Please visit https://docs.snowplowanalytics.com/docs/contributing/contributor-license-agreement/ to learn more and sign.

Once you've signed, please reply here (e.g. I signed it!) and we'll verify. Thanks.

File: provisioning/resources/configs/Caddyfile [hunk 1]
```diff
@@ -38,7 +43,11 @@
     }
 
     route /nsqadmin* {
-        reverse_proxy localhost:4171
+        reverse_proxy localhost:4171 {
+          header_down +Content-Security-Policy "default-src 'self'; frame-ancestors 'none'"
+          header_down +X-Frame-Options "DENY"
+        }
+        header_down +X-Frame-Options "DENY"
```
Comment (istreeter) [inline provisioning/resources/configs/Caddyfile:50]: whoops this line shouldn't be here.

File: provisioning/resources/ui/js/components/Overview.tsx [hunk 2]
```diff
@@ -53,8 +53,8 @@ export class Overview extends React.Component<{}, {}> {
         <h3>The software stack installed: </h3>
         <ul>
         <li><b>Snowplow Mini 0.13.6</b></li>
-          <li>Snowplow Stream Collector NSQ 2.5.0</li>
-          <li>Snowplow Stream Enrich NSQ 2.0.5</li>
+          <li>Snowplow Stream Collector NSQ 3.0.0</li>
```
Comment (oguzhanunlu) [inline provisioning/resources/ui/js/components/Overview.tsx:56]: unexpected collector version bump here

**Review by oguzhanunlu — APPROVED** [review]
Thanks Ian!

**Pull request closed without merging**